packages/shared/src/components/onboarding/ReadingReminder.tsx

```diff
@@ -22,10 +22,13 @@ const ReadingReminderOptions = [
 ];
 
 interface ReadingReminderProps {
-  onClickNext: () => void;
+  onClickNext: (skipped?: boolean) => void;
```

Comment:
This is hard to recognize when is used, maybe we can convert into this
```suggestion
  onClickNext: (options?: {skipped?: boolean}) => void;
```

Reply:
Agreed. I'll update it.